examples/cloudrun/analysis/service.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       containers:
       - args:
         - server
-        image: ghcr.io/pipe-cd/helloworld:v0.30.0
+        image: gcr.io/pipecd/piped:v0.27.4
```

Comment:
not sure why do we use piped image here instead of helloworld image 🤔

Reply:
Oops. My bad, they must be helloworld.

Reply:
@khanhtc1202 Fixed them!